Backend/Middleware/src/main/java/middleware/util/ValidationUtil.java

```diff
@@ -0,0 +1,52 @@
+package middleware.util;
+
+import middleware.model.JWToken;
+
+/**
+ * ValidationUtil provides utility methods for validating JWT session data, UUIDs, and user roles.
+ * These methods are used to ensure that the JWT and its claims are consistent with the data stored in the database.
+ */
+public class ValidationUtil {
+
+    /**
+     * Validates the JWT session by checking if the JWT exists in the database.
+     *
+     * @param dbJwt the JWT object retrieved from the database.
```

Comment:
this is like a small thing but for things with capitralised references like UUID and JWT i think we can keep them capitalised dunnid camelcase

Reply:
Honestly I did it more so uk where the next word start but makes sense imma change wait a sec

Reply:
Act I'll change it to retrievedJWT